tests/functional/adapter/test_grants.py

```diff
@@ -111,7 +111,7 @@ def snapshots(self):
 
 
 class TestInvalidGrants(BaseInvalidGrants):
-    # SingleStore creates the user and sets a warning:
+    #   SingleStore creates the user and sets a warning:
```

Comment:
you can add `SET GLOBAL NO_AUTO_CREATE_USER = 1` to you CREATE ROLES command, then you'll be able to run the test with 8.1 as well.